### PR TITLE
Feat login 160453644

### PR DESCRIPTION
**Pull request by Ginnynjau**
The login page that will enable the User to access for my To-do app.
This is the solution for task ID#160453644.
 
@codjoero, @lubwamabenja, @mwinel, @nuwagalb @citytycoon, I request your reviews.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            </div>
+        </div>
+    </header> 
+
+    <!--Show-->
+    <section id="show">
+        <div class="container">
+            <div class="row middle-xs middle-sm middle-md middle-lg">
+                <div class="col-xs-12 col-sm-12 col-md-12 col-lg- show-content">
+                    <marquee> <p>  The <span class="primary-text">Ultimate </span> Task list Generator</p> </marquee>
```
Comment (mwinel) [inline index.html:32]: Are you working on making your list items scroll horizontally?

**Review by mwinel — COMMENTED** [review]
Your login "submit" should redirect to the user profile as the user story states.

File: index.html (hunk 1 above)
Comment (Ginnynjau) [reply index.html:32]: That is just for a catch phrase scrolling on the page. It is for decoration.

Comment (Ginnynjau) [reply index.html:32]: As for the submit button, that is not how forms work. The facilitator advised us just to create the forms and we will work on the back end without php as we continue.

**Comment by codjoero** [comment]
Its a nice simple site... Just suggesting the body cover the whole page real estate, which puts the footer information at the bottom of the page.

**Comment by Ginnynjau** [comment]
> Its a nice simple site... Just suggesting the body cover the whole page real estate, which puts the footer information at the bottom of the page.

Thanks for the suggestion.

**Comment by lubwamabenja** [comment]
when you actually minimize your log in page to the finest ,the input fields
overlap
so i think there is a problem with your media queries
thanks

**Comment by nuwagalb** [comment]
@Ginnynjau am joining the party late but can you provide your gh-page link so that I can test to see if your page is responsive

**Comment by lubwamabenja** [comment]
https://github.com/lubwamabenja/TODOLIST

**Pull request merged**